.travis.yml

```diff
@@ -1,5 +1,8 @@
 osx_image: xcode8.2
 language: objective-c
+branches:
+  only:
+    - master
```

Comment:
@wkiefer fyi. added this and re-enabled travis building for pushes so now it will build for all pull requests and for pushes (merges) to master. But travis won't be triggered for a push to a non-master branch (which was causing all our pull requests to have two queued travis builds).